test/integration/bitswap_wo_routing_test.go

```diff
@@ -19,22 +17,15 @@ func TestBitswapWithoutRouting(t *testing.T) {
 	const numPeers = 4
 
 	// create network
-	mn, err := mocknet.FullMeshLinked(ctx, numPeers)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	peers := mn.Peers()
-	if len(peers) < numPeers {
-		t.Fatal(errors.New("test initialization error"))
-	}
```

Comment:
why removing this? it can catch serious errors that might otherwise drive people mad.